components/ItemCard/ItemCard.js

```diff
@@ -0,0 +1,66 @@
+// Import external resources and hooks
+import styled from "styled-components";
+import { useState } from "react";
+
+// Import of styled components
+import * as Styled from "@/components/ItemCard/ItemCard.styled.js";
+
+// Import of SVG Icon Components
+import * as Icon from "@/components/Icons";
+
+export function ItemCard({ product, setCartTotal, cartTotal }) {
```

Comment:
```suggestion
export function ItemCard({ product, onSetCartTotal, cartTotal }) {
```

Reply:
- [x] Fix: replaced it with handle function